multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/ProcessTypeVariablesUpdater.java

```diff
@@ -0,0 +1,32 @@
+package org.cloudfoundry.multiapps.controller.process.util;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.cloudfoundry.multiapps.controller.api.model.ProcessType;
+import org.cloudfoundry.multiapps.controller.process.steps.ProcessContext;
+import org.cloudfoundry.multiapps.controller.process.variables.Variables;
+
+@Named
+public class ProcessTypeVariablesUpdater {
+
+    private ProcessTypeParser processTypeParser;
+
+    @Inject
+    public ProcessTypeVariablesUpdater(ProcessTypeParser processTypeParser) {
+        this.processTypeParser = processTypeParser;
+    }
+
+    public void updateProcessTypeVariables(ProcessContext context) {
```

Comment:
The name can be shorten to update()
This is because the name of the class which hints that process type variables will be updated